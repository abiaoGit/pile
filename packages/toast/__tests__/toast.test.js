import { render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';

describe('test toast component', () => {
//   it('snapshot ui', () => {
//     const demo = require('../../../site/src/docs/toast.md').default;
//     const wrapper = render(demo);
//     expect(renderToJson(wrapper)).toMatchSnapshot();
//   });
  it('test 1', () => {
    expect(true).toEqual(true);
  });
});
