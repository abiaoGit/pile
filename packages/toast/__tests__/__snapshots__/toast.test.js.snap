// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test toast component snapshot ui 1`] = `null`;
